model/service_logs/v1/cluster_logs_uuid_resource.model

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright (c) 2022 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages the collection of cluster logs by custer uuid.
+resource ClusterLogsUUID {
+	// Retrieves the list of cluster logs by cluster uuid.
+	method List {
+		// Index of the requested page, where one corresponds to the first page.
+		in out Page Integer = 1
+
+		// Maximum number of items that will be contained in the returned page.
+		in out Size Integer = 100
+
+		// Total number of items of the collection that match the search criteria,
+		// regardless of the size of the page.
+		out Total Integer
+
+		// Order criteria.
+		//
+		// The syntax of this parameter is similar to the syntax of the _order by_ clause of
+		// a SQL statement. For example, in order to sort the
+		// cluster logs descending by name identifier the value should be:
+		//
+		// ```sql
+		// name desc
+		// ```
+		//
+		// If the parameter isn't provided, or if the value is empty, then the order of the
+		// results is undefined.
+		in Order String
+
+		// Search criteria.
+		//
+		// The syntax of this parameter is similar to the syntax of the _where_ clause
+		// of an SQL statement, but using the names of the attributes of the cluster logs
+		// instead of the names of the columns of a table. For example, in order to
+	        // retrieve cluster logs with service_name starting with my:
+		//
+		// ```sql
+		// service_name like 'my%'
+		// ```
+		//
+		// If the parameter isn't provided, or if the value is empty, then all the
+		// items that the user has permission to see will be returned.
+		in Search String
+
+		// Retrieved list of Cluster logs.
+		out Items []LogEntry
+	}
+
+    // Reference to the service that manages a specific Log entry.
+	locator LogEntry {
+		target LogEntry
+		variable ID
+	}
```

Comment:
I this really supported? I think there are no sub-resources of this resource.

Reply:
LogEntry is a class in `cluster_log_type.model`

Reply:
This locator translates into a `/api/service_logs/v1/clusters/{cluster_id}/cluster_logs/{entry_id}` endpoint. I think we don't have such an endpoint in the server, do we? If we don't have that endpoint then this locator can and should be removed.

Reply:
I see .. ok I'll remove that one. thanks